packages/compass-indexes/src/components/indexes-toolbar/indexes-toolbar.tsx

```diff
@@ -1,30 +1,32 @@
 import React, { useCallback } from 'react';
 import { connect } from 'react-redux';
 import {
-  withPreferences,
   usePreference,
+  withPreferences,
 } from 'compass-preferences-model/provider';
 import {
   Button,
-  ErrorSummary,
-  Tooltip,
-  WarningSummary,
-  Link,
   css,
-  spacing,
+  DropdownMenuButton,
```

Comment:
Just a small note for the future, I'm not sure what's doing the re-ordering of the imports here, we don't have this style requirement in compass, but it makes reviewing this code way harder because it's not clear what exactly changed in the imports. If it's possible at all not to do, that would be very helpful 🙂